cmd/create/cluster/cmd.go

```diff
@@ -871,27 +899,39 @@ func run(cmd *cobra.Command, _ []string) {
 		args.ec2MetadataHttpTokens = string(v1.Ec2MetadataHttpTokensOptional)
 	}
 
+	// Errors when users elects for cluster admin via flags and elects for hosted control plane via interactive prompt"
+	if isHostedCP && (createAdminUser || createAdminPassword != "") {
+		r.Reporter.Errorf("Setting Cluster Admin is only supported in classic ROSA clusters")
+		os.Exit(1)
+	}
+
+	// isClusterAdmin is aflag indicating if user wishes to create cluster admin
```

Comment:
```suggestion
	// isClusterAdmin is a flag indicating if user wishes to create cluster admin
```

Reply:
Edited via sep. commit. Thank you for the find!